packages/collections/graphql/schema-types.js

```diff
@@ -45,11 +45,17 @@ extend type Document {
   userProgress: DocumentProgress
 }
 
+enum ProgressState {
+  FINISHED
+  UNFINISHED
+}
```

Comment:
My intuition tells me, one state is missing. WDYT?

```suggestion
enum ProgressState {
  NOT_STARTED
  IN_PROGRESS
  COMPLETED
}
```

Reply:
`NOT_STARTED` state or deleted positions (`removeDocumentProgress`) are represented by no record being present. We only start recording the position once someone scrolled over the first paragraph of the main text.

Reply:
API !== implementation

Reply:
Ok. I get it now.